db/db.go

```diff
@@ -0,0 +1,79 @@
+package db
+
+import (
+	"context"
+	"fmt"
+
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+const (
+	monthResultsCollection = "month-results"
+	dbName                 = "dadosjusbr"
+)
+
+//MonthResults is a data model of the results of one month parsing
+type MonthResults struct {
+	Month           int
+	Year            int
+	SpreadsheetsURL string
+	DatapackageURL  string
+	Status          string //success or fail (put in consts??)
+}
+
+//Client manages all iteractions with mongodb
+type Client struct {
+	client *mongo.Client
+}
+
+//NewClient returns an db connection instance that can be used for CRUD opetations
+func NewClient(url string) (*Client, error) {
+	// Set client options
+	clientOptions := options.Client().ApplyURI(url)
+
+	// Connect to MongoDB
+	client, err := mongo.Connect(context.TODO(), clientOptions)
+	if err != nil {
+		return nil, err
+	}
+
+	// Check the connection
+	err = client.Ping(context.TODO(), nil)
```

Comment:
Pode colocar essa linha na linha de baixo. So colocar dentro do if (antes do teste) e separar por ";" .. Como num for.

Reply:
Não sabia que dava pra fazer isso. Achei massa :)